packages/cloud-cognitive/src/components/Datagrid/Datagrid.stories.js

```diff
@@ -35,9 +35,10 @@ import {
   useDisableSelectRows,
   useCustomizeColumns,
   useSelectAllWithToggle,
+  useColumnCenterAlign,
 } from '.';
 
-import useColumnCenterAlign from './useColumnCenterAlign';
+// import useColumnCenterAlign from './useColumnCenterAlign';
```

Comment:
Can you remove this line entirely (now that it is imported with the other hooks above)?